pyproject.toml

```diff
@@ -192,20 +192,16 @@ exclude = [
 [tool.ruff.pydocstyle]
 convention = 'numpy'
 
-[tool.pytest]
-python_files = [
-    'benchmark_*.py',
-    'test_*.py',
-]
-python_classes = [
-    'Test*',
-    '*Suite',
-]
-python_functions = [
-    'time_*',
-    'test_*',
-    'peakmem_*',
-]
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "-ra  --strict-config  --strict-markers"
+log_cli_level = "INFO"
+xfail_strict = true
+#testpaths = ["skimage", "../../../../../benchmarks"]
```

Comment:
Our benchmark suite is broken right now when calling pytest on it. We are treating our benchmarks very stepmotherly right now. I intend to enable this option, once we have fixed the suite.

Some random thoughts on this: The approach currently taken by `spin test` makes it more difficult to run pytest on anything that is not included in the the [`f"--rootdir={site_path}"`](https://github.com/scientific-python/spin/blob/2b928bae17461748c6c5eed564dcf77a81775cd8/spin/cmds/meson.py#L228). That may apply to tests that are not included in the distribution archive, or in our case our benchmark suite. So we get this awkward inclusion path: `../../../../../benchmarks`. In this context I played around with [meson-python's (>= 0.14.0) new editable install](https://meson-python.readthedocs.io/en/latest/how-to-guides/editable-installs.html#editable-installs). That approach seems to make a lot of `spin`'s magic unnecessary. So I am carefully optimistic that in the future we can make our dev setup and tooling a lot less confusing a simpler. :crossed_fingers: